lib/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,35 @@
+export interface ToastProps {
+  /** Text shown on the toast */
+  text: string;
+  /** Icon shown on the toast */
+  icon?: string;
+  /** Aria label for the icon */
+  iconAriaLabel?: string;
+  variant?: 'success' | 'warning' | 'error';
```

Comment:
Maybe would be good to tell somewhere in the Storybook if not in JSDoc, that color is not enough to communicate the "priority" vs other variants so icon should be a must.

Reply:
Fixed now